apps/nextjs-website/react-components/components/PressReleaseList/PressReleaseList.helpers.tsx

```diff
@@ -17,7 +35,9 @@ export const PressReleasePreview = ({
 
   return (
     <Stack gap={2} width={330}>
-      <Typography variant='overline' color='text.secondary'>{date}</Typography>
+      <Typography variant='overline' color='text.secondary'>
+        {formatDateToLocale(date, locale)}
```

Comment:
```suggestion
        {date}
```
Already formatted in makePressReleaseListProps